vhdbuilder/packer/components.json

```diff
@@ -340,8 +340,6 @@
       ],
       "multiArchVersions": [
         "v0.11.0.2",
-        "v0.11.1",
```

Comment:
0.11.1 is higher than 0.11.0.2

Reply:
cool, so I will remove "v0.11.0.2", sounds ok?